packages/components/src/components/hive-footer.tsx

```diff
@@ -26,7 +26,9 @@ export function HiveFooter({ className, logo, resources = [], sameSite }: HiveFo
             {...logo}
             className="hive-focus -m-1.5 flex rounded p-1.5"
           >
-            <HiveCombinationMark className="h-8 w-auto text-green-1000 dark:text-white" />
+            {logo?.children || (
```

Comment:
href should be changeable too, e.g. in codegen when you click on logo in footer it should go to root of codegen cc @saihaj

Reply:
you can change `href`, as it's already part of `logo: ILink`, and the default here should be `href="/"`

Reply:
ah yeah your right